modules/fluid_properties/include/materials/FluidPropertiesMaterialPH.h

```diff
@@ -0,0 +1,38 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "Material.h"
+#include "SinglePhaseFluidProperties.h"
+
+/**
+ * Computes fluid properties using (pressure, temperature) formulation
+ */
+class FluidPropertiesMaterialPH : public Material
```

Comment:
I added the tests.
It s to be able to access these properties from the input file.
I could go either way, but there's already one checked in the repo as a normal object I think.

Reply:
yeah Ben only added PH. PT and VE (without suffix) were already in the repo
I think they're convenient

Reply:
Ok, it looks like these are intended for output only, not in any residual objects, since there are no derivatives.

- I think the VE and PT ones make some sense, since these variables are often solution variables. PH, not so much - does anyone actually want to use this?
- If these are used for output, then I say: why stop at the quantities that are directly computable by that variable set? I'd think the objective is to compute everything that you can (all of the quantities that `SinglePhaseFluidProperties` offers. Right now I notice `FluidPropertiesMaterialVE` and `FluidPropertiesMaterialPT` compute different sets of quantities, for example.

Reply:
Note that `FluidPropertiesInterrogator` is doing the same thing as these materials, but of course these materials have the advantage of being able to do it at QPs rather than a single point. It'd probably be a nice refactor at some point to have some kind of thermodynamic state definition class (where all quantities would be computed, given some variable set), but I definitely won't ask for that here.

Reply:
I ll move PH to a test object. 
I ll see if we can add more, but there's already a lot I would not be surprised if it s already maxed out

Reply:
You have `T = T(p,h)` as your first call, so then you have `(p,T)` and thus everything. However, it's not really useful to add more things unless it's a regular (non-test) object.

Reply:
well I like to be able to see the variation, but realistically I dont know who works fully with (p, h) and would require this. We only have 2 calls which are (p,h). 

I m thinking we dont actually want too many properties too. The more properties we have the more likely these materials may not be used with a fluid property UO, because of an unimplemented property